Add storage details (#1249)
platformsh/platformsh-docs: src/configuration/app/storage.md

## Disk

Your _plan storage size_ specifies the maximum total space available to _all_ applications and services.

> When deploying your project, the sum of all `disk` keys define din `.platform.app.yaml` and `.platform/services.yaml` 

Reviewer: ```suggestion
When deploying your project, the sum of all `disk` keys defined in `.platform.app.yaml` and `.platform/services.yaml` 
```

must be *equal or less* than the _plan storage size_. For example, if your _plan storage size_ is 5 GB, you can assign:

- 2 GB to your application, 3 GB to your database
- 1 GB to your application, 4 GB to your database

> - 1 GB to your application, 1 GB to your database, 3 GB to your Elasticsearch service and so on.

Reviewer: ```suggestion
- 1 GB to your application, 1 GB to your database, 3 GB to your Elasticsearch service
- etc.
```

Reviewer: Also, use asterisks rather than dashes for the bullets.  That's what we've been doing in most of the docs.

If you receive an error on `git push` mentioning the total disk space configured for the application and its services exceeds the plan storage size, you need to either increase the disk space reserved for your project on the project setup page or lower the storage assigned to each service and the application.

The `disk` key is required, and defines the size of the persistent disk of the application (in MB).  Its minimum value is 256 MB and a validation error will occur if you try to set it lower.

> 

Reviewer: ```suggestion
```
